src/h5web/visualizations/line/ErrorBarCurve.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Suspense, ReactElement } from 'react';
+import { LineSegments } from 'react-three-fiber/components';
+import { useDataGeometry } from './hooks';
+
+const DEFAULT_COLOR = '#1b998b';
+
+interface Props {
+  abscissas: number[];
+  ordinates: number[];
+  errors: number[];
+  color?: string;
+}
+
+function ErrorBarCurve(props: Props): ReactElement {
+  const { abscissas, ordinates, errors, color = DEFAULT_COLOR } = props;
+
+  const errorAbscissas = [];
+  const errorOrdinates = [];
+  for (let index = 0; index < abscissas.length; index++) {
+    // [x, y - yerr]
+    errorAbscissas.push(abscissas[index]);
+    errorOrdinates.push(ordinates[index] - errors[index]);
+
+    // [x, y + yerr+]
+    errorAbscissas.push(abscissas[index]);
+    errorOrdinates.push(ordinates[index] + errors[index]);
+  }
+
+  const errorGeometry = useDataGeometry(errorAbscissas, errorOrdinates);
+
+  return (
+    <Suspense fallback={<></>}>
```

Comment:
Might as well put `null` as fallback.

Reply:
Actually, I think I can remove the `Suspense` completely as there is no loading of external resources (contrary to the sprites in `DataCurve`)